webpages/settings/style.css

```diff
@@ -132,9 +132,13 @@ h1 {
 
 .theme-switch {
   height: 25px;
-  right: 25px;
+}
+#theme-div {
+  right: 0;
   position: fixed;
   cursor: pointer;
+  padding: 16px;
+  transition: 0.2s ease;
```

Comment:
There isn't anything to transition to right now although a hover background would be nice.

Reply:
Ah!, i copy pasted that code from #7214 which had a hover effect

Reply:
I'll remove that for now, and add it back when i create a pr for hover background (which will first require this pr to be merged)